### PR TITLE
Optimizing, updating docs

### PR DESCRIPTION
**Pull request by rkononov**
1) Changed the way how explorer grab the blocks, optimized algorithm
2) Updated readme, docs
3) Updated titles, labels

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,9 +0,0 @@
-## Ethereum Classic Explorer License
```
Comment (treeder) [inline LICENSE.md]: Have to keep the license.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,4 @@
-# GoChain Block Explorer
-
-Simple interface for exploring the GoChain blockchain.
+# GochainExplorer
```
Comment (treeder) [inline README.md]: Keep what it was, or at least two words. And add back the description.

**Comment by rkononov** [comment]
@treeder updated

**Comment by treeder** [comment]
Build failed.

**Comment by rkononov** [comment]
@treeder 
1) npm install was failing because of merge with master - https://github.com/gochain-io/explorer/blob/master/package.json#L2
2) Tests fixed, added Makefile, circle config and so on (followed the same way of launching tests as was used in original repo) - in future would be nice to replace everything with mocks

**Pull request merged**